outlines/generate/generator.py

```diff
@@ -55,7 +58,8 @@ def sequence_generator(
     fsms: List["FSM"],
     init_state: Tuple,
     fsm_states: List[FSMState],
-    rng: torch.Generator,
+    num_samples: int = 1,
```

Comment:
While I think `sequence_generator` should be able to take inputs of any shape, I am not sure it should be aware of `num_samples`.